code/modules/mob/living/simple_animal/hostile/cockroach.dm

```diff
@@ -66,6 +66,11 @@
 		if(isliving(AM))
 			var/mob/living/A = AM
 			if(A.mob_size > MOB_SIZE_SMALL && !(A.movement_type & FLYING))
+				if(ishuman(A))
```

Comment:
Why the human check? Is monky with pacifism not allowed to be nice to cockroaches?

Reply:
Good point

Reply:
i didn't know monkeys could have traits